libr/core/panels.c

```diff
@@ -2342,7 +2342,7 @@ static bool init(RCore *core, RPanels *panels, int w, int h) {
 	panels->isResizing = false;
 	panels->can = createNewCanvas (core, w, h);
 	panels->db = sdb_new0 ();
-	panels->mht = ht_new ((DupValue)strdup, (HtKvFreeFunc)mht_free_kv, (CalcSize)strlen);
+	panels->mht = ht_new (NULL, (HtKvFreeFunc)mht_free_kv, (CalcSize)strlen);
```

Comment:
I think you have to keep this strdup

Reply:
It has to be removed, since the value on the hashtable is not a string (it's a pointer to a panel item, I removed because it was trying to duplicate the item pointer as if it was a string)

Reply:
oh i see, lgtm